tests/qmctl-test/test_qmctl.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+echo "Running qmctl command..."
+
+QMCTL_SCRIPT="../../tools/qmctl/qmctl"
+
+if [ ! -f "$QMCTL_SCRIPT" ]; then
+  echo "Error: qmctl script not found at $QMCTL_SCRIPT"
+  exit 1
+fi
+
+sudo dnf install qm -y
```

Comment:
Please set this test to run under 
/plans/e2e/kvm-tier-0
It is running setup automatically

Add this tag here  filter: 'tag:qmctl-test' to kvm-tier-0 discovery

Reply:
Added the filter